src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,165 @@
+                service.register(user));
+        user.setLogin("u125R");
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+    }
+
+    @Test
+    void register_equalsOrMoreThanSixLoginLength_ok() {
+        user.setLogin("Eva123");
+        user.setPassword("123456");
+        Storage.people.add(user);
+        assertTrue(Storage.people.contains(user));
+        Storage.people.clear();
+        user.setLogin("01Eva123");
+        Storage.people.add(user);
+        assertTrue(Storage.people.contains(user));
+    }
+
+    @Test
+    void register_nullPassword_notOk() {
+        user.setLogin("123456ID");
+        user.setPassword(null);
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+    }
+
+    @Test
+    void register_lessThanSixPasswordLength_notOk() {
+        user.setLogin("12tr3456");
+        user.setPassword("12");
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+        Storage.people.clear();
+        user.setLogin("6372wv34");
+        user.setPassword("123");
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+        Storage.people.clear();
+        user.setLogin("836128de");
+        user.setPassword("12345");
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+    }
+
+    @Test
+    void register_equalsOrMoreThanSixPasswordLength_ok() {
+        user.setLogin("Make123");
+        user.setPassword("012345");
+        Storage.people.add(user);
+        assertTrue(Storage.people.contains(user));
+        Storage.people.clear();
+        user.setLogin("01Dana123");
+        user.setPassword("012345");
+        Storage.people.add(user);
+        assertTrue(Storage.people.contains(user));
+    }
+
+    @Test
+    void service_nullAge_notOk() {
+        user.setLogin("12Make123");
+        user.setPassword("ds012345");
+        user.setAge(null);
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+    }
+
+    @Test
+    void service_AgeLessThanEighteenth_notOk() {
+        user.setLogin("1ab3426");
+        user.setPassword("423457");
+        user.setAge(-20);
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+        Storage.people.clear();
+        user.setLogin("1272ew53");
+        user.setPassword("12345423");
+        user.setAge(0);
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+        Storage.people.clear();
+        user.setLogin("8wa36128");
+        user.setPassword("44434512");
+        user.setAge(11);
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+    }
+
+    @Test
+    void service_AgeEqualsOrMoreThanEighteenth_ok() {
+        user.setLogin("Olly563");
```

Comment:
you need only one _ok test, where you test user all properties of which are valid
don't use storage clear() in your tests, you have afterEach already, this shouldn't be necessary